### PR TITLE
adds a number (4) of tgs commands

### PR DESCRIPTION
**Pull request by harryob**
## About The Pull Request

adds @[botname] check, gameversion, testmerges and (admin only) status

## Why It's Good For The Game

uhh you can check some game data without being in game

## Changelog

:cl:
server: added new discord commands: check, gameversion, testmerges and status (for admins). check what these do with the help command
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
dont game ver & testmerges kinda double up?

**Comment by harryob** [comment]
> dont game ver & testmerges kinda double up?

CleanShot 2022-11-24 at 13 50 57@2x
i figured the top two lines are kinda useless data if someone's just looking for testmerges - but i'm not particularly sold either way, can be ditched to avoid bloating our commands

File: code/modules/discord/tgs_commands.dm [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	\n**Gamemode:** [gamemode]"
+	return(status)
+
+/datum/tgs_chat_command/check
+	name = "check"
+	help_text = "Gets the playercount, gamemode, and address of the server"
+
+/datum/tgs_chat_command/check/Run(datum/tgs_chat_user/sender, params)
+	var/server = CONFIG_GET(string/server)
+	var/message = "\
```
Comment (Zonespace27) [inline code/modules/discord/tgs_commands.dm:29]: same here

File: code/modules/discord/tgs_commands.dm [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				msg += "PR #[PR.number] at [copytext_char(PR.head_commit, 1, 9)] [PR.title].\n"
+				if (PR.url)
+					msg += "<[PR.url]>\n"
+	return msg.Join("")
+
+/datum/tgs_chat_command/testmerges
+	name = "testmerges"
+	help_text = "Gets the current TMs"
+
+/datum/tgs_chat_command/gameversion/Run(datum/tgs_chat_user/sender, params)
```
Comment (fira) [inline code/modules/discord/tgs_commands.dm:67]: harry you didn't reanme the proc when copypasting it

**Review by fira — CHANGES_REQUESTED** [review]

**Pull request closed without merging**